examples/getting_started/quickstart.ipynb

```diff
@@ -0,0 +1,626 @@
+    ")\n",
+    "def stream_capital_info(country: str):\n",
+    "    pass\n",
+    "\n",
+    "\n",
+    "for chunk, _ in stream_capital_info(\"France\"):\n",
+    "    print(chunk.content, end=\"\", flush=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this example:\n",
+    "\n",
+    "1. We enable streaming mode with the `stream=True` parameter in the `@openai.call` decorator.\n",
+    "2. Our prompt asks for detailed information about a capital city, which is likely to generate a longer response.\n",
+    "3. The response is returned in small chunks that can be processed immediately.\n",
+    "4. We use `end=\"\"` and `flush=True` to display the output immediately without line breaks, creating a smooth streaming effect.\n",
+    "\n",
+    "This approach has several advantages:\n",
+    "\n",
+    "1. **Responsiveness**: Users see the beginning of the response immediately, rather than waiting for the entire response to be generated.\n",
+    "2. **Handling Long Outputs**: For detailed information like this, streaming allows you to start processing or displaying the response before it's fully generated.\n",
+    "3. **Real-time Interaction**: You could potentially process the chunks as they come in, allowing for real-time analysis or formatting of the response.\n",
+    "\n",
+    "For example, you could modify the processing to format the output in real-time:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import re\n",
+    "\n",
+    "\n",
+    "def format_capital_info(chunk: str) -> str:\n",
+    "    # Convert \"History:\" (and similar headers) to bold\n",
+    "    chunk = re.sub(r\"(\\w+):\", r\"**\\1:**\", chunk)\n",
+    "    # Add bullet points to lists\n",
+    "    chunk = re.sub(r\"^(\\s*)(\\d+\\.)\", r\"\\1- \", chunk, flags=re.MULTILINE)\n",
+    "    return chunk\n",
+    "\n",
+    "\n",
+    "for chunk, _ in stream_capital_info(\"Japan\"):\n",
+    "    formatted_chunk = format_capital_info(chunk.content)\n",
+    "    print(formatted_chunk, end=\"\", flush=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This modification formats the streamed content in real-time, bold-ing headers and converting numbered lists to bullet points. This demonstrates how streaming can be combined with real-time processing to enhance the presentation of information as it's generated.\n",
+    "\n",
+    "Streaming is particularly useful when dealing with longer, more detailed responses, such as comprehensive information about capital cities. It allows for a more interactive and responsive user experience, especially in applications where immediate feedback is valuable."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Response Models\n",
```

Comment:
as I mentioned in my comment re: tools, I think the `Response Models` section should start with an extremely simple example where the goal is just to get structured information.

We can include additional information about validators if we think we should, but we can also just link to the learn section if we think that would be better?